### ISSUE TITLE
Doesnt not bind to IPv6 addresses when http disabled (UDP Only)

### ISSUE THREAD

**Issue by dtrackd** [feross/bittorrent-tracker#73]
When running this:

```
bittorrent-tracker -p 80 --http false -q
```

Seems that the tracker only binds to the IPv4 addresses (`0.0.0.0`) which is fine but does not bind to IPv6 address (`[::]`)

```
root@box:~# lsof -i :80
COMMAND   PID USER   FD   TYPE  DEVICE SIZE/OFF NODE NAME
node    32129 root   10u  IPv4 1580383      0t0  UDP *:http
```

Here you can see on IPv4 its listening ok:
```
root@box:~#  nmap -sU -p 123.123.123.123

Starting Nmap 6.00 ( http://nmap.org ) at 2015-05-02 07:33 MSK
Nmap scan report for 123.123.123.123
Host is up.
PORT   STATE         SERVICE
80/udp open|filtered http

Nmap done: 1 IP address (1 host up) scanned in 2.19 seconds
```

and for IPv6 udp port is closed:
```
root@box:~# nmap -6 -sU 2a00:dead::beef -p 80

Starting Nmap 6.00 ( http://nmap.org ) at 2015-05-02 07:33 MSK
Nmap scan report for 2a00:dead::beef
Host is up (0.000050s latency).
PORT   STATE  SERVICE
80/udp closed http

Nmap done: 1 IP address (1 host up) scanned in 0.08 seconds
```

**Comment by feross** [comment]
Fixed and released as 3.6.0.

**Comment by dtrackd** [comment]
Sorry about another comment on this, I think you forget to deal with line 10 on ``parse_udp.js``

```
...
if (rinfo.family !== 'IPv4') throw new Error('udp tracker does not support IPv6')
...
```

Currently in the tracker I am getting ``WARNING: udp tracker does not support IPv6``

(sorry about all these reports, I can imagine it starting to get annoying) 

Cheers

**Comment by feross** [comment]
Sorry about that. Fixed in 3.6.1.

No worries - your reports are helping to make this module more robust. They're appreciated.
